2765. Longest Alternating Subarray/solution.py

```diff
@@ -0,0 +1,31 @@
+class Solution:
+    def alternatingSubarray(self, nums: List[int]) -> int:
+        # define the length of the longest subarray starting at 1
+        
+        # define the current subarray length starting at 1
+        
+        # define the previous number starting at the first number
+        
+        # for each number that's not the first...
+        
+            # the subarray is supposed to look like [3,4,3,4,3,...], 
+            # so s[i+1] - s[i] = (-1)^(i+1). 
+            
+                # if it meets the requirements, make the previous
+                # number this number and increment the current
```

Comment:
what do you mean make the previous number this number? Are you incrementing an index?

Reply:
Ah, ok I saw your code. Better to save, set the previous number variable to the current number.

Reply:
Or for the comment when you declare the "previous number", make it clear that that variable is to store the value of the number before the current number so you can diff them.